velox/expression/tests/ExpressionVerifier.cpp

```diff
@@ -68,45 +68,39 @@ void compareVectors(const VectorPtr& left, const VectorPtr& right) {
   LOG(INFO) << "All results match.";
 }
 
-RowVectorPtr makeRowVector(const VectorPtr& vector) {
-  return std::make_shared<RowVector>(
-      vector->pool(),
-      ROW({vector->type()}),
-      nullptr,
-      vector->size(),
-      std::vector<VectorPtr>{vector});
-}
-
 } // namespace
 
 ResultOrError ExpressionVerifier::verify(
-    const core::TypedExprPtr& plan,
+    const std::vector<core::TypedExprPtr>& plans,
     const RowVectorPtr& rowVector,
     VectorPtr&& resultVector,
     bool canThrow,
     std::vector<column_index_t> columnsToWrapInLazy) {
-  LOG(INFO) << "Executing expression: " << plan->toString();
-
+  for (int i = 0; i < plans.size(); ++i) {
+    LOG(INFO) << "Executing expression " << i << " : " << plans[i]->toString();
+  }
   logRowVector(rowVector);
 
   // Store data and expression in case of reproduction.
   VectorPtr copiedResult;
-  std::string sql;
+  std::string sql = "";
 
-  // Complex constants that aren't all expressable in sql
+  // Complex constants that aren't all expressible in sql
   std::vector<VectorPtr> complexConstants;
   // Deep copy to preserve the initial state of result vector.
   if (!options_.reproPersistPath.empty()) {
     if (resultVector) {
       copiedResult = BaseVector::copy(*resultVector);
     }
-    std::vector<core::TypedExprPtr> typedExprs = {plan};
-    // Disabling constant folding in order to preserver the original
-    // expression
+    // Disabling constant folding in order to preserve the original expression
     try {
-      sql = exec::ExprSet(std::move(typedExprs), execCtx_, false)
-                .expr(0)
-                ->toSql(&complexConstants);
+      auto exprs = exec::ExprSet(plans, execCtx_, false).exprs();
+      for (int i = 0; i < exprs.size(); ++i) {
+        if (i > 0) {
+          sql += ", ";
```

Comment:
The reason i am using a comma is because thats how this can be fed back into ExpressionRunner and the parser relies on this comma to separate out expression trees.